nav2_bringup/package.xml

```diff
@@ -9,28 +9,10 @@
   <license>Apache License 2.0</license>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
-
-  <build_depend>rclcpp</build_depend>
-  <build_depend>navigation2</build_depend>
-  <build_depend>builtin_interfaces</build_depend>
-  <build_depend>rosidl_default_generators</build_depend>
-  <build_depend>launch_ros</build_depend>
-
-  <exec_depend>launch_ros</exec_depend>
-  <exec_depend>rclcpp</exec_depend>
-  <exec_depend>builtin_interfaces</exec_depend>
-  <exec_depend>rosidl_default_runtime</exec_depend>
+  
+<exec_depend>launch_ros</exec_depend>
   <exec_depend>navigation2</exec_depend>
 
-  <test_depend>ament_lint_common</test_depend>
```

Comment:
Please don't delete the test depends or the general rclcpp/launch_ros in any package. if someone wants to add a test, launch file, or new node they're needed and frequently forgotten.

The others are OK. Though I think `navigation` should also stay since bringup means nothing without it